src/plugins/borrower-profile-exp-mixin.js

```diff
@@ -61,16 +57,6 @@ export default {
 				logReadQueryError(e, 'borrowerProfileSideSheetQuery');
 			});
 		},
-		initializeIsBpModalEnabledExp(category) {
```

Comment:
Change file name? As no more an experiment?

Reply:
Long-term I want to consolidate this file into a composable, but I did think about renaming...